common/types.go

```diff
@@ -16,6 +16,18 @@ var (
 	// ErrForbidden means the user doesn't have access to this resource.
 	ErrForbidden = errors.New("forbidden")
 
+	// ErrInvalidSessionId means the session ID is invalid.
+	ErrInvalidSessionId = errors.New("invalid session id")
+
+	// ErrUnableToLockRow means the resource couldn't be locked.
+	ErrUnableToLockRow = errors.New("unable to lock row")
```

Comment:
I wonder if this should just be `ErrRetryable` instead? Since it is easily retryable and will likely work in a few seconds after the lock is gone.